Adds instructions to deploy locally without Cloud Build. (#105)
amandakarina/terraform-example-foundation: 4-projects/README.md

### Run Terraform locally

1. The next instructions assume that you are at the same level of the `terraform-example-foundation` folder. Create and change into `gcp-projects` folder, copy the code, Terraform wrapper script and ensure it can be executed.

   ```bash
   mkdir gcp-projects
   cp -R  terraform-example-foundation/4-projects/* gcp-projects
   cp terraform-example-foundation/build/tf-wrapper.sh gcp-projects/
   cp terraform-example-foundation/.gitignore gcp-projects
   chmod 755 ./gcp-environments/tf-wrapper.sh
   ```

> 1. Navigate to `gcp-projects` and initialize a local git repository, so you can manage versions locally. Create the environment branches.

Reviewer: ```suggestion
1. Navigate to `gcp-projects` and initialize a local Git repository to manage versions locally. Then, create the environment branches.
```
